### DISCUSSION TITLE
File sharing between VM and host

### DISCUSSION THREAD

**Discussion by lowrisk75** [utmapp/UTM#4713] [General]
I have a debian VM on my MacMini m1
I did install the following, but still have no clipboard or shared folder
apt install spice-vdagent
apt install spice-webdavd

I need to store file from this VM to a folder on my host 

I tried this but that didn't work
$ sudo mkdir /media/share/
$ sudo mount -t 9p -o trans=virtio share /media/share/ -oversion=9p2000.L

it's return 
/media/share: bad option; for several filesystems (e.g. nfs, cifs) you might need a /sbin/mount.<type> helper program.

same with 
root@homeassistant:~# mount -t virtiofs share /media/share/
mount: /media/share: wrong fs type, bad option, bad superblock on share, missing codepage or helper program, or other error.

**Comment by dingsen2** [comment]
Hi, did you resolve this issue?

**Comment by dturaev** [comment]
Happened to me after cloning a VM (not sure if this was the cause, though). Clearing and re-adding the shared directory solved it.

**Reply by TheFern2** [reply]
This worked for me, I was going crazy. I setup a vm, then cloned it and couldn't figure out why the file shared folder worked on my base vm, but not the clone.
